app/src/main/res/values/strings.xml

```diff
@@ -404,6 +408,10 @@
         ]]>
     </string>
 
+    <string name="lessons_help_input_phrase">
+        TODO
```

Comment:
Will it be added later?

Reply:
Oops, thanks, I’ll add it tomorrow and then merge

Reply:
I decided to use the same help message as for `InputSymbol` fragments